packages/ui/src/views/repo/repo-summary/components/edit-repo-details-dialog.tsx

```diff
@@ -0,0 +1,90 @@
+import { ChangeEvent, useState } from 'react'
+
+import {
+  AlertDialog,
+  AlertDialogContent,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  Button,
+  ButtonGroup,
+  Icon,
+  Text,
+  Textarea
+} from '@/components'
+
+interface EditRepoDetailsDialog {
+  showEditRepoDetails: boolean
+  description?: string
+  onSave: (desc: string) => void
+  onClose: () => void
+  updateRepoError?: string
+  isSubmitting: boolean
+  isSubmitted: boolean
+}
+
+export const EditRepoDetails = ({
+  showEditRepoDetails,
+  description,
+  onSave,
+  onClose,
+  updateRepoError,
+  isSubmitting,
+  isSubmitted
+}: EditRepoDetailsDialog) => {
+  const [newDesc, setNewDesc] = useState<string>(description || '')
+  return (
+    <AlertDialog open={showEditRepoDetails} onOpenChange={onClose}>
+      <AlertDialogContent className="h-80 max-h-[70vh] max-w-[460px]" onOverlayClick={onClose}>
+        <AlertDialogHeader>
+          <AlertDialogTitle className="mb-4">Repository Description</AlertDialogTitle>
+        </AlertDialogHeader>
+        <Text className="inline-block leading-none text-foreground-7" size={1}>
```

Comment:
```suggestion
        <span className="inline-block leading-none text-foreground-7 text-14">
```

Let’s try to avoid using the `Text` component. We recently discussed in Slack that we should phase it out in the future since it doesn’t serve much purpose inside components. Instead, you can confidently use the appropriate HTML tags directly.